src/Sarif.Multitool.Library/MergeCommand.cs

```diff
@@ -173,14 +175,17 @@ private async Task<bool> MergeSarifLogsAsync()
 
                                 if (!_ruleIdToRunsMap.TryGetValue(key, out Run splitRun))
                                 {
-                                    splitRun = _ruleIdToRunsMap[key] = new Run()
-                                    {
-                                        Tool = emptyRun.Tool,
-                                        Results = new List<Result>()
-                                    };
+                                    emptyRun.Results = new List<Result>();
+                                    splitRun = _ruleIdToRunsMap[key] = emptyRun;
```

Comment:
>emptyRun [](start = 71, length = 8)

Instead of creating a new run, I changed to emptyRun object to preserve artifacts and other fields